### DISCUSSION TITLE
希望增加HAProxy负载均衡

### DISCUSSION THREAD

**Discussion by hululu1068** [Openwrt-Passwall/openwrt-passwall2#156] [Ideas]
HAProxy负载均衡对于不好的线路，有非常大的作用，不知道下个版本会不会添加这个神器

**Comment by handsomeji** [comment]
我已将大部分设备转移到passwall2，此功能可考虑一下。

**Comment by handsomeji** [comment]
https://github.com/xiaorouji/openwrt-passwall2/commit/d8e1004b8f43dcd6c87f45ca18bf8c63ec9f260a
